default/python/test/AI/Behavior/freeOrionAIInterface.py

```diff
@@ -0,0 +1,20 @@
+# Fake fo for testing Behavior purposes
+
+# TODO Find a better way to import the static portions of fo without the test chaos, overhead and random state of fo.
```

Comment:
This is mock for autocomplete. It is importable.   You can take `aggression` from there
https://github.com/freeorion/freeorion/blob/master/default/AI/freeorion_debug/ide_tools/result/freeOrionAIInterface.py